visual_testing_wdio/test/pageObjects/Page.js

```diff
@@ -0,0 +1,26 @@
+/* global resolutionBreakPoint */
+class Page {
+
+    get navigationBar() {
+        return $('#header-outer')
+    }
+
+    get navigationButton() {
+        return $('//span[@class="close-wrap loaded"]')
```

Comment:
This is also not a button? 
If you need this, you can use:
`//a[@aria-label="Navigation Menu"]`